metagpt/logs.py

```diff
@@ -34,15 +31,15 @@ def define_log_level(print_level="INFO", logfile_level="DEBUG", name: str = None
 logger = define_log_level()
 
 
-def log_llm_stream(msg):
+def log_llm_stream(msg: str) -> None:
+    """Log LLM stream messages."""
     _llm_stream_log(msg)
 
 
-def set_llm_stream_logfunc(func):
+def set_llm_stream_logfunc(func: Callable[[str], None]) -> None:
+    """Set the function to be used for logging LLM streams."""
     global _llm_stream_log
     _llm_stream_log = func
 
 
-def _llm_stream_log(msg):
-    if _print_level in ["INFO"]:
```

Comment:
It looks like this changes have reverted the feature introduced in PR #1124, which allowed the `define_log_level` method to control stream logging via the `print_level` parameter.  Could you please clarify the reason for this change?